<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QLabel" name="bgFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>621</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/greeny.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/minivector.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LblTxt1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Scanner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/minivector.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LblTxt2">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>218</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Dataset&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/minivector.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LblTxt3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Health&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/minivector.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LblTxt4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Train IDS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Month">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#55ffff;&quot;&gt;March&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Userphoto">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>60</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/myPic2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="jarvisGIF">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>240</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/circle.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameAI">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/resources/lwFrame.jpg);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#ffff7f;&quot;&gt;AstraeA-v1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btnexit">
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>499</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 255);
background-image: url(:/rec/resources/electric1.png);
font: 75 24pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/jarvisheart.jpg</normaloff>:/rec/resources/jarvisheart.jpg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>470</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/minivector.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 24pt &quot;Monospace&quot;;
color: rgb(246, 245, 244);
background-image: url(:/rec/resources/electric1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/jarvisheart.jpg</normaloff>:/rec/resources/jarvisheart.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 24pt &quot;Monospace&quot;;
color: rgb(246, 245, 244);
background-image: url(:/rec/resources/electric1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/jarvisheart.jpg</normaloff>:/rec/resources/jarvisheart.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 24pt &quot;Monospace&quot;;
color: rgb(246, 245, 244);
background-image: url(:/rec/resources/electric1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/jarvisheart.jpg</normaloff>:/rec/resources/jarvisheart.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 24pt &quot;Monospace&quot;;
color: rgb(246, 245, 244);
background-image: url(:/rec/resources/electric1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/jarvisheart.jpg</normaloff>:/rec/resources/jarvisheart.jpg</iconset>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDdate">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(52, 226, 226);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDyear">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(52, 226, 226);</string>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>2021</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>799</x>
     <y>280</y>
     <width>235</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Sysinfo">
    <item>
     <widget class="QLabel" name="Sysno">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;User: 18CS1143&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Syspermit">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Permisssion: User&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Sysip">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;IP: 192.168.42.173&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Sysmbps">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Network: 2mbps&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btnmail">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>498</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 24pt &quot;Monospace&quot;;
color: rgb(246, 245, 244);
background-image: url(:/rec/resources/electric1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/resources/jarvisheart.jpg</normaloff>:/rec/resources/jarvisheart.jpg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rec/resources/minivector.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LblTxt5">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>518</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Mail&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LblTxt6">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>519</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55ffff;&quot;&gt;Exit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Footerlogo">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 255);
font: 20pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>BY CYBERKID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Helpbutton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;Arial Black&quot;;
background-image: url(:/rec/resources/lwFrame.jpg);
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <zorder>bgFrame</zorder>
  <zorder>Lbl1</zorder>
  <zorder>Lbl2</zorder>
  <zorder>Lbl3</zorder>
  <zorder>Lbl4</zorder>
  <zorder>Month</zorder>
  <zorder>Userphoto</zorder>
  <zorder>jarvisGIF</zorder>
  <zorder>nameAI</zorder>
  <zorder>Lbl5</zorder>
  <zorder>Btnexit</zorder>
  <zorder>Btn1</zorder>
  <zorder>LblTxt1</zorder>
  <zorder>Btn2</zorder>
  <zorder>LblTxt2</zorder>
  <zorder>Btn3</zorder>
  <zorder>LblTxt3</zorder>
  <zorder>Btn4</zorder>
  <zorder>LblTxt4</zorder>
  <zorder>LCDdate</zorder>
  <zorder>LCDyear</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>Lbl6</zorder>
  <zorder>Btnmail</zorder>
  <zorder>LblTxt5</zorder>
  <zorder>LblTxt6</zorder>
  <zorder>Footerlogo</zorder>
  <zorder>Helpbutton</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
